xbmc/cores/VideoPlayer/VideoRenderers/OverlayRenderer.cpp

```diff
@@ -577,28 +601,24 @@ void CRenderer::Notify(const Observable& obs, const ObservableMessage msg)
 
 void CRenderer::LoadSettings()
 {
-  m_subtitleHorizontalAlign = static_cast<SubtitleHorizontalAlign>(
-      CServiceBroker::GetSettingsComponent()->GetSettings()->GetInt(
+  m_subtitleHorizontalAlign =
+      static_cast<SubtitleHorizontalAlign>(SUBTITLES::CSubtitlesSettings::GetSettings()->GetInt(
```

Comment:
Hum my suggestion was to extend the `CSubtitleSettings` class with your access methods:

```cpp
SubtitleHorizontalAlign CSubtitlesSettings::GetHorizontalAlignment () 
{
   return static_cast<SubtitleHorizontalAlign>(m_settings->GetInt(CSettings::SETTING_SUBTITLES_CAPTIONSALIGN));
}
```

Same for other access methods. At the moment the class is a singleton (but only used by OverlayRenderer) but maybe at some point (depending on who can access it) it can be even made a simple member var)

Reply:
done, moved all settings in CSubtitlesSettings